src/bscript.js

```diff
@@ -84,6 +86,21 @@ class BScript {
     }
   }
 
+  get redeemScript () {
+    if (this.tokens.length === 1 && this.tokens[0].type === Token.LITERAL) {
```

Comment:
Why does this.tokens.length have to be 1, and in what cases would it not be?

Reply:
The script must be a p2sh scriptSig, which is always a single PUSH_DATA op.  That is why the length must be 1 and have type `Token.LITERAL`.